CheatSheets/html-cheatsheet.md

```diff
@@ -144,3 +151,19 @@
 | `<col>`                    | Denotes a column inside a table.                                   |
 
 **[🔼Back to Top](#table-of-contents)**
+
+## Graphics
+
+| Command                    | Description                                                        |
+| `<canvas> … </canvas>`     | Used to draw graphics on a web page using javascript.              |
+| `<svg> … </svg>`           | Used to defines vector-based graphics in XML format.               |
+
+**[🔼Back to Top](#table-of-contents)**
+
+## Graphics
+
+| Command                    | Description                                      |
+| `<video> … </video>`       | Used to show a video on a web page.              |
+| `<audio> … </audio>`       | Used to play an audio file on a web page.        |
+
+**[🔼Back to Top](#table-of-contents)**
```

Comment:
Those lines of code `break the table format`.
- Double `Graphics header`
- Double `canvas` added